dotCMS/src/main/java/com/dotcms/security/multipart/MultiPartSecurityRequestWrapper.java

```diff
@@ -182,13 +182,15 @@ private void checkSecurityInputStream(final InputStream tmpStream) throws IOExce
         }
     }
 
+    private final static String[] checkPatterns  = {"content-disposition","form-data","filename"};
+
     private void testString(final String lineToTest) {
 
         final String lineToTestLower = lineToTest.toLowerCase();
-
-        if (!lineToTestLower.contains("content-disposition:") || ! lineToTestLower.contains("filename=")) {
-
-            return;
+        for (String p : checkPatterns) {
```

Comment:
u can do Stream.of(checkPatterns).map(String::toLowerCase).filter(pattern -> lineToTestLower.contains(pattern)).anyMatch()

something like that

Reply:
I was thinking that the compiler/JIT optimizations could easily inline a String array loop